.github/main.workflow

```diff
@@ -3,8 +3,15 @@ workflow "Build & deploy docs" {
   resolves = ["Publish Documentation"]
 }
 
+action "Only Master" {
```

Comment:
Can this be https://github.com/actions/bin/tree/master/filter instead?

Reply:
Aha! Yes.